model/Packager/ZipPackager.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA ;
+ */
+
+namespace oat\taoSync\model\Packager;
+
+
+use oat\oatbox\service\ConfigurableService;
+use oat\taoSync\model\Exception\PackagerException;
+
+class ZipPackager extends ConfigurableService implements PackagerInterface
```

Comment:
I'm a bit concerned about the fact that `ZipPackager` is service and we have only one instance of that class.

In case if it will be used by two different services it may lead to unexpected issues.

Maybe that makes sense to not register that class as a service but configure export service with a class name and create new instance each time when `ExportService::export()` called?